README.md

```diff
@@ -52,16 +52,25 @@ $ npm install jasmine-enzyme --save-dev
 
 ## Setup
 
-###### Jest
-If you are using [jest](https://facebook.github.io/jest/), the simplest setup is to use jests `setupTestFrameworkScriptFile` config. Add this to your `package.json`
+### Jest
+If you are using [jest](https://facebook.github.io/jest/), the simplest setup
+is to use jest's `setupTestFrameworkScriptFile` config. You'll also need to
+tell jest to unmock `react`, `enzyme`, and `enzyme-jest`.
+
+Make sure your `package.json` includes the following:
 
 ```js
 "jest": {
   "setupTestFrameworkScriptFile": "node_modules/jasmine-enzyme/lib/jest.js",
-}
+  "unmockedModulePathPatterns": [
+    "<rootDir>/node_modules/react/",
+    "<rootDir>/node_modules/enzyme",
+    "<rootDir>/node_modules/jasmine-enzyme"
```

Comment:
a bit nit picky. Can you remove the parts about rootDir and node_modules? Jest uses these as regex strings, so just having the package name suffices. If you think it's better to have the explicitness, I'm not super against it. But maybe remove the `/` from the react line so it's consistent.